.env_SAMPLE

```diff
@@ -35,6 +35,7 @@ export WORDPRESS=http://www.consumerfinance.gov
 #export HTTP_HOST=localhost
 #export HTTP_PORT=7000
 #export SELENIUM_URL=http://localhost:4444/wd/hub
+#export TOXENV=py27
```

Comment:
At some point it would probably make sense to add some code comments in here for the less obvious variables. Doesn't have to happen here.